### DISCUSSION TITLE
IssueCertificate failed. No DNS Entries

### DISCUSSION THREAD

**Discussion by infosys-dape** [polymind-inc/acmebot#580] [Q&A] [answered]
Hey, im struggeling since a few day with my Key Vault Acmebot. 

I hope someone can help me here. 

I created a test DNS on Azure and followed the instructions to install the Bot on azure. 

Now i got the following error:

![image](https://user-images.githubusercontent.com/118541945/230040459-4c4177ec-0b8a-4b66-b97e-7b466f2e1c2b.png)

i already tried to restart the function app. I also tried to give the rights for network contributor. In the least attempt i tried to create a new dns zone. But i  always get the same error. Im pretty sure followed the instructions right. Something seems to be wrong with the rights. All ressources are an the same ressource group. 

Do someone have any idea what i can try else? 

Thank you very much in advance

**Comment by shibayan** [comment, marked as answer]
In addition to creating the resource in Azure DNS, the domain must be delegated to Azure DNS. Please read the following tutorial.

https://learn.microsoft.com/en-us/azure/dns/dns-delegate-domain-azure-dns#delegate-the-domain

**Comment by infosys-dape** [comment]
Hey thank you very much. My bad. Such a simple thing and my brain just passed out haha. 

Thank you very much, that did the fix
